ext/RMagick/rmfill.c

```diff
@@ -207,11 +207,11 @@ vertical_fill(
     }
 
     // Now copy the master row to each actual row.
-    for (y = 0; y < image->rows; y++)
+    for (y = 0; y < (ssize_t) image->rows; y++)
```

Comment:
unnecessary type cast.
```suggestion
    for (y = 0; y < image->rows; y++)
```